### DISCUSSION TITLE
所有设备全部掉线

### DISCUSSION THREAD

**Discussion by einsbiao** [XiaoMi/ha_xiaomi_home#265] [Q&A]
![微信截图_20241220091354](https://github.com/user-attachments/assets/0739de2f-d443-4ff6-ade4-79fe544f3032)
如图所示，所有设备突然全部掉线，按照说明更新设备列表后一段时间内可用，过十几分钟还是会突然全部掉线。

**Comment by SusanPhevos** [comment]
您好！请提一个issue，并附上问题复现时的Home Assistant日志。
